examples/3-twitter-clone/src/app/models/MessageModel.ts

```diff
@@ -38,7 +38,7 @@ export const MessageModel = MessageModelBase.views(self => ({
     },
     loadReplies() {
       if (!loadReplyQuery) {
-        loadReplyQuery = self.store.loadMessages(0, 100, self.id)
+        loadReplyQuery = self.store.loadMessages("", 100, self.id)
```

Comment:
`loadMessages` expects a string as first argument instead of a number. So I needed to change it. Don't know how it could compile correctly before 🤔 and mostly I don't know if I changed it correctly. However, I tried the demo and it works.

Reply:
in `examples/3-twitter-clone/src/app/models/RootStore.ts` at line `54`, `""` is passed to `loadMessages()` so I assume I fixed it correctly